addons/mail/models/mail_thread.py

```diff
@@ -3062,6 +3062,7 @@ def _message_auto_subscribe_notify(self, partner_ids, template):
             model_description = self.env['ir.model']._get(record._name).display_name
             values = {
                 'object': record,
+                'company': self.env.company,
```

Comment:
If that's only for the colors, why not a helper that returns the colors and avoid adding company to the rendering envionment ?